exercises/resistor-color-trio/canonical-data.json

```diff
@@ -60,6 +60,78 @@
         "value": 470,
         "unit": "kiloohms"
       }
+    },
+    {
+      "uuid": "ee7a1b79-8848-4b1a-bf50-498c6fab7f27",
+      "description": "Brown and red and red",
+      "property": "label",
+      "input": {
+        "colors": ["brown", "red", "red"]
+      },
+      "expected": {
+        "value": 1200,
+        "unit": "ohms"
+      }
+    },
+    {
+      "uuid": "5f6404a7-5bb3-4283-877d-3d39bcc33854",
+      "description": "Blue and violet and grey",
+      "property": "label",
+      "input": {
+        "colors": ["blue", "violet", "grey"]
+      },
+      "expected": {
+        "value": 6700,
+        "unit": "megaohms"
+      }
+    },
+    {
+      "uuid": "7d3a6ab8-e40e-46c3-98b1-91639fff2344",
+      "description": "Minimum possible value",
+      "property": "label",
+      "input": {
+        "colors": ["black", "black", "black"]
+      },
+      "expected": {
+        "value": 0,
```

Comment:
A bit unsure of this one. 
A lot of the zero ohms resistor is just one black band.
Sure there are some with multiple but the most common ones have just 1 band.

Reply:
These resistors, be exercise definition, all have three bands. Other resistors which do not meet the exercise description are irrelevant. This exercise isn't asking students to read all real world resistors.